tests/behat/features/operate-subsites.feature

```diff
@@ -28,7 +28,7 @@ Feature: Create and select a subsite
     And I press the "Add folder" button
     And I select "Main site" from "SubsitesSelect"
     # Using a short folder name so that it doesn't get truncated on the frontend
-    And I fill in "Folder name" with "mfol"
+    And I fill in "Form_folderCreateForm_Name" with "mfol"
```

Comment:
For some reason we now need to get the same field via ID rather than name

Reply:
Uhhh that feels bad. Did the name of the form field change or something? This feels like it could lead to unexpected regressions, e.g. if someone has js or css relying on form field names?

Reply:
I don't think it changed, behat was just fussy for some reason

Reply:
If nothing changed, nothing needs to change here.

But evidently you've had to change something here, which means something _did_ change. Behat being fussy doesn't explain it, because this change isn't necessary in the current `4` branch, it's only needed when applying the changes in these PRs.

So _something_ you've changed has caused this to need to change, and that's scaring me a little that we don't know what it was or what other side effects that might have.

Reply:
I'll try changing it back and see if it's resolved, does work locally, we'll see if CI likes it

Reply:
Works with it reverted on latest kitchen sink run